client/js/admin.js

```diff
@@ -43,6 +44,18 @@ var submit = function() {
 	var content = editor.getValue()
 	var tags = $("#tags").val().split(" ")
 	var title = $("#title").val()
+
+	var deleting = false
+
+	if (!(content || tags.length > 1 || tags[0] || title)) {
```

Comment:
It shouldn't really make a difference, because the idea is that they remove the entire content to delete, but I can roll that tin